docs/Deploy-Ergo-Node-via-Jenkins.md

```diff
@@ -0,0 +1,63 @@
+Deploy Ergo Node to Testnet or Mainnet via Jenkins
+==================================================
+
+This manual describes how you can deploy specific build of Ergo node to Testnet or Mainnet.
+
+
+## Features and limitations
+
+- Ergo node `master` branch automatically deployed to Testnet after integration tests successfully done
+- There is no automatic deploy to Mainnet of any kind
+- You should manually follow instructions below to deploy to Mainnet, as to deploy specific (not `master`) branch to Testnet
+- You can not deploy builds older than 60 days
+- [Integration tests](https://jenkins.ergoplatform.com/job/ergo-it/) must be successfully done for deploying build, at least, `sbt` compilation must ended up successfully. But it is strongly recommended never deploy failed builds
+- Pass congiguration files with deploy is not supported now. Because of this, all nodes in Mainnet and Testnet has as general configs as possible. Configs files are placed under [files/configs/ergo-testnet](../files/configs/ergo-testnet) for Testnet and config file template [files/configs/ergo-mainnet](../files/configs/ergo-mainnet/application.conf.j2) for Mainnet and periodically update via Ansible. Feel free to [create issue](https://github.com/ergoplatform/ansible/issues/new) to add ability pass configs with deploys
```

Comment:
```suggestion
- Passing of configuration files to deploy is not supported now. Because of this, all nodes in Mainnet and Testnet has as general configs as possible. Configs files are placed under [files/configs/ergo-testnet](../files/configs/ergo-testnet) for Testnet and config file template [files/configs/ergo-mainnet](../files/configs/ergo-mainnet/application.conf.j2) for Mainnet and periodically update via Ansible. Feel free to [create issue](https://github.com/ergoplatform/ansible/issues/new) to add ability pass configs with deploys
```